### PR TITLE
PR template introduction

### PR DESCRIPTION
**Pull request by AhmetCanSolak**
With this PR, I am introducing a PR template to keep a standard on our development process and make things easier for possible future contributor friends.

### CODE REVIEW

**Comment by jni** [comment]
Some general comments: in my opinion, less is more with PR templates. We have a much shorter list in scikit-image and it has had quite a limited effect on PR quality or even checklist compliance, as far as I can tell. I think the *only* thing that is absolutely required is complete test coverage of any changes, together with measurements from continuous integration systems (so that is the first thing that I would set up).

In contrast, the style guide (which you allude to, but needs to exist before this template is adopted) and contributing guide are very important. People don't usually read them ahead of time, but (a) having CONTRIBUTING.md means they get a big yellow warning banner when submitting a PR, and (b) once people submit a PR, it's very quick to prod them to comply by pointing them to specific clauses in the contributing guide. This kind of personal ping is much more successful than the template.

I suggest you look at the scikit-image contributing guide and template, and maybe even copy the former wholesale. As I mentioned, I think the template should be minimized: even the one we have in scikit-image is too much in my opinion.

http://scikit-image.org/docs/dev/contribute.html

**Comment by AhmetCanSolak** [comment]
Thank you for your detailed comment @jni !
I will be updating with a new commit.

I and @kne42 were also planning/writing a contributing guideline, will drop it in another PR.

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+[Tell us about your new feature, improvement, or fix! Fixes # (issue)]
+
+## Type of change
+[Please delete options that are not relevant.]
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## How Has This Been Tested?
```
Comment (kne42) [inline PULL_REQUEST_TEMPLATE.md:11]: I don't think that this section is necessary. Reviewers need only look through the files to find the proper test cases (which they should be doing in the first place).

Comment (AhmetCanSolak) [reply PULL_REQUEST_TEMPLATE.md:11]: I agree with you on short-run but might be useful when there are many people contributing.

**Review by kne42 — APPROVED** [review]
LGTM!

**Pull request merged**